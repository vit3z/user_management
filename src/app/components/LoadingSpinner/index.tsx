import styles from "./index.module.css";

const LoadingSpinner = () => {
  return <div className={styles.userManagementLoader}></div>
}

export default LoadingSpinner;