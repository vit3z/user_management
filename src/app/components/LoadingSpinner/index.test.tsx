import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import LoadingSpinner from '.'
 
describe("Loading Spinner", () => {
  it('renders the loading spinner in a snapshot', () => {
    const { container } = render(<LoadingSpinner />)
    expect(container).toMatchSnapshot()
  })
  it('renders the spinner', () => {
    const { container } = render(<LoadingSpinner />);
    expect(container).toBeInTheDocument();
  })
})