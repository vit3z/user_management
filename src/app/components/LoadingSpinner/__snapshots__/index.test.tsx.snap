// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Spinner renders the loading spinner in a snapshot 1`] = `
<div>
  <div
    class="userManagementLoader"
  />
</div>
`;
